bin/profile-merge-log

```diff
@@ -42,7 +42,7 @@ const logProfileMerges = async () => {
         continue;
       }
 
-      const mergeTime = task.updatedAt.toISOString();
+      const mergeTime = typeof task.updatedAt === 'object' ? task.updatedAt.toISOString() : task.updatedAt;
```

Comment:
`typeof null === 'object'`, so depending on the root cause this may not fix your problem.

```suggestion
      const mergeTime = task.updatedAt instanceof Date ? task.updatedAt.toISOString() : task.updatedAt;
```